lib/Sema/CSBindings.cpp

```diff
@@ -1045,6 +1045,13 @@ bool TypeVariableBinding::attempt(ConstraintSystem &cs) const {
     cs.DefaultedConstraints.push_back(srcLocator);
 
     if (type->isHole()) {
+      if (auto * OLE = dyn_cast<ObjectLiteralExpr>(srcLocator->getAnchor())) {
```

Comment:
Nit: Could you please make it a last in the `else if` chain and use `dyn_cast_or_null` to be safe?

Reply:
Also `*OLE` shouldn't have a space after `*`.

Reply:
Sure :)